modules/ROOT/pages/tools/io-tool/filesending.adoc

```diff
@@ -0,0 +1,46 @@
+:imagesdir: 
+= File Sending
+
+When you share, you can choose to publish or send the files. Publishing means you send them out and anyone who is interested and has access can download them. On the other hand, when you send the files, you select a specific recipient.
```

Comment:
When sending a file ... who is interested ("subscribed") will receive a copy" (not "has access to download them")

Reply:
"On the other hand, when you send the files, you select a specific recipient."

The sentence doesn't sound good, "On the other hand" is not right here and a description on how to choose an endpoint and which endpoint to choose would be important. Would be an own subtitled chapter.

You can linke the concept of publish and subscribe from the docs (see build/site/agrirouter-interface-documentation/latest/communication.html)